api/resolvers/search.js

```diff
@@ -195,7 +195,14 @@ export default {
       const { query, quotes, nym, url } = queryParts(q)
 
       if (url) {
-        whatArr.push({ match_phrase_prefix: { url: `${url.slice(4).toLowerCase()}` } })
+        const domain = url.startsWith('url:www.') ? url.slice(8) : url.slice(4)
```

Comment:
While OP wrote title as "disregard www. unless search included www." I thought widening the scope to other subdomains better satisfied OP's intention of "wanting a wider net when searching on a domain name only".